### PR TITLE
corrected formatting in dataviz part 1

### PR DESCRIPTION
**Pull request by mukulrawat1986**
Corrected the formatting in the dataviz tutorial part 1, there is a big jumbled mesh in between the text which was corrected.

### CODE REVIEW

**Comment by econchick** [comment]
Thanks @mukulrawat1986 ! Would you mind making it as one commit, rather than 5?  You can do that easily by rebasing ([howto here][https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/]) then force-pushing to your repo again; it'll auto-update the PR.  Once that's done I'll review all the changes.

**Comment by mukulrawat1986** [comment]
I have squashed the commits into one, i hope this works. Thanks @econchick

File: website/_containers/dataviz/2013-01-04-part-1.md [hunk 1]
```diff
@@ -117,25 +116,25 @@ So we’ve told Python to open the file, now we have to read the file. We have t
...
-Here, `csv.reader` is a function of the CSV module. We gave it two parameters: opened_file, and delimiter. It’s easy to get confused when parameters and variables share names. In `delimiter=delimiter`, the first 'delimiter' is referring to the name of the parameter that `csv.reader` needs; the second 'delimiter' refers to the argument that our `parse` function takes in.
+Here, <code>csv.reader</code> is a function of the CSV module. We gave it two parameters: opened_file, and delimiter. It’s easy to get confused when parameters and variables share names. In <code>delimiter=delimiter</code>, the first <code>delimiter</code> is referring to the name of the parameter that <code>csv.reader</code> needs; the second <code>delimiter</code> refers to the argument that our <code>parse</code> function takes in.
 
-Just to quickly put these two lines in our `parse` function:
+Just to quickly put these two lines in our <code>parse</code> function:
 
 ```python
 def parse(raw_file, delimiter):
-	"""Parses a raw CSV file to a JSON-line object"""
+  """Parses a raw CSV file to a JSON-line object"""
```
Comment (merwok) [inline website/_containers/dataviz/2013-01-04-part-1.md]: Could you use 4-space indents for code?

**Comment by econchick** [comment]
Yeah in line with what @merwok mentioned, that last code example you edited, there was a reason for all the spacing - to show the scope of the function. And it's typical/PEP8 compliant to have 4 rather than 2 spaces.

**Comment by mukulrawat1986** [comment]
Thanks @merwok and @econchick , i have corrected the indents in the code.

**Comment by ProProgrammer** [comment]
Hey, I was going through dataviz part 1 tutorial and I noticed the formatting is still improper. i was wondering when this commit will be merged into master.

**Comment by econchick** [comment]
Thanks for the nudge, @ProProgrammer !  I'll look into reviewing & merging it this weekend

**Comment by ProProgrammer** [comment]
:+1:

**Comment by mukulrawat1986** [comment]
Hey @econchick  can you review and merge the commits

**Comment by econchick** [comment]
I can't if it's closed :) sorry for the delay and thanks for the reminder.

**Comment by mukulrawat1986** [comment]
@econchick sorry, i didn't know that i closed it. I have reopened it.

**Comment by econchick** [comment]
Merged & deployed - thanks for the ping and the pull request!

**Pull request merged**